frontend/src/widgets/OverviewWidgetField.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Tooltip as TrussWorksToolTip } from '@trussworks/react-uswds';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faQuestionCircle,
+} from '@fortawesome/free-solid-svg-icons';
+import './OverviewWidgetField.scss';
+
+import Tooltip from '../components/Tooltip';
+import colors from '../colors';
+
+export function OverviewWidgetField({
+  label1,
+  label2,
+  route,
+  data,
+  icon,
+  iconColor,
+  backgroundColor,
+  showTooltip,
+  tooltipText,
+  filterApplicable,
+  iconSize,
+}) {
+  const smallIconClass = iconSize === 'sm' ? 'smart-hub--dashboard-overview-widget-field-icon__background-sm' : 'smart-hub--dashboard-overview-widget-field-icon__background-lg';
```

Comment:
do we use a different icon size at any point? I only notice because I forked this branch to work on filters and this line doesn't have test coverage. Not aware of the need for two sizes but maybe I'm forgetting something

Reply:
I removed the class if statement for test coverage.

Reply:
Removed for coverage.